Doc/whatsnew/3.9.rst

```diff
@@ -121,6 +121,11 @@ Added the *indent* option to :func:`~ast.dump` which allows it to produce a
 multiline indented output.
 (Contributed by Serhiy Storchaka in :issue:`37995`.)
 
+Added the :func:`ast.unparse` as a function in the :mod:`ast` module that can
```

Comment:
Mentioning, it is migrated from `Tools/parser/unparse.py` would be cool to people who are familiar with this tool.

Reply:
I don't think this is necessary, people can always use git history to know where it comes from.